lib/plugins/EPrints/Plugin/Import/PubRouter.pm

```diff
@@ -153,14 +138,23 @@ sub input_fh_xml
 
 	my $doc = EPrints::XML::parse_xml_string( $xml );
 
-	my $dataobj = $plugin->xml_to_dataobj( $opts{dataset}, $doc->documentElement );
-		
-	EPrints::XML::dispose( $doc );
-	
-	return EPrints::List->new(
-       	dataset => $opts{dataset},
-       	session => $plugin->{session},
-       	ids => [defined($dataobj) ? $dataobj->get_id : ()] );
+	my $node = $doc->documentElement;
+	my @ns = $node->getNamespaces;
+	my $pr_uri = $node->lookupNamespaceURI( "pr" );
+	if( $pr_uri ne "http://pubrouter.jisc.ac.uk/rioxxplus/v2.0/" )
+	{
+		die "Wrong version of XML received, expecting xmlns:pr=\"http://pubrouter.jisc.ac.uk/rioxxplus/v2.0/\" got xmlns:pr=\"$pr_uri\”";
```

Comment:
Sorry, just spotted this, which I think might cause unexpected result:
In the last string ....`xmlns:pr=\"$pr_uri\”";` it looks like the second escaped quote is different from the first escaped quote.

Reply:
@adam-rehin No worries - I think that is harmless, but it would have been annoying! Should be fixed in commit https://github.com/jisc-services/eprints-plugin-PubRouter-RIOXXplus-Connector/pull/10/commits/ea398bc9283ee610cc0ded5ec280b55b95f15431